src/commands/reward/index.ts

```diff
@@ -69,9 +69,10 @@ const reward = async (
   }
 
   // Find linked issue assignees.
-  const issueNumber = findLinkedIssueNumber(data.body);
+  const issueMsg = findLinkedIssueNumber(data.body);
+  const issueNumber = issueMsg[0];
```

Comment:
I think you can give it a name. Not directly use the index.

Reply:
Sorry, I didn't understand. Issue MSG is a tuple。I want to get the first element

Reply:
const [firstName,secondName] = findLinkedIssueNumber(data.body);

Reply:
ok, I understand.

Reply:
I'm worried. secondName, I didn't use it.

Reply:
You can ignore the second value.

Reply:
I solved the problem